Gemfile

```diff
@@ -159,6 +159,9 @@ group :production do
   gem 'unicorn-worker-killer', require: false
 end
 
+# pin i18n version while we patch it
+# cf. lib/open_project/patches/i18n-reject-empty-string.rb
+gem 'i18n', '~> 1.8.8'
```

Comment:
I don't think we need to pin it if we force the patch to be checked upon update as done within the patch file.